.agents/skills/populate-jtbds/SKILL.md

```
@@ -272,15 +272,15 @@ For each confirmed `<object>#<relation>` group:
      delegate type's relation block even if the `object_type` also exists in
-     `model.yaml`.
+     `model.fga`.
 2. **Replace** only the `# @fgadoc:jtbd` lines in the comment block
    immediately above that `define`. Leave all other lines untouched.
 3. The comment format must be:
 
-```yaml
-            # @fgadoc:jtbd <statement one>
-            # @fgadoc:jtbd <statement two>
-            define <relation>: ...
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)